apps/prairielearn/assets/scripts/navbarClient.ts

```diff
@@ -5,42 +5,43 @@ import CookiesModule from 'js-cookie';
 
 import { onDocumentReady } from '@prairielearn/browser-utils';
 
+// Keep in sync with src/lib/client/cookies.ts
 const COOKIE_EXPIRATION_DAYS = 30;
 
-onDocumentReady(() => {
```

Comment:
The functions were reordered/pulled out of the `onDocumentReady`, but the onDocumentReady did not go away.

Reply:
These changes were pulled into #12431 , so don't review this PR yet.